.github/workflows/qunit_tests-additional-renovation.yml

```
@@ -0,0 +1,186 @@
+name: Additional QUnit
+
+concurrency:
+  group: wf-${{github.event.pull_request.number}}-${{github.workflow}}
+  cancel-in-progress: true
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build
+    timeout-minutes: 60
```

Maybe we can decrease the timeout so that the build will be aborted earlier in case if it stuck
```suggestion
    timeout-minutes: 15
```